PayloadPkg/FirmwareUpdate/GetCapsuleImage.c

```diff
@@ -166,7 +166,7 @@ GetCapsuleFromRawPartition (
     return EFI_NOT_FOUND;
   }
 
-  if (FwUpdHeader->PubKeySize != RSA_MOD_SIZE + RSA_E_SIZE + sizeof (UINT32)) {
+  if (FwUpdHeader->PubKeySize != RSA_MOD_SIZE_2048 + RSA_E_SIZE + sizeof (UINT32)) {
```

Comment:
RSA_MOD_SIZE_2048 is only supported? What about RSA_MOD_SIZE_3072?

Reply:
I have restricted this patch for only modifying interfaces and hard coding removals in secure libs.
These should be taken care while adding support for RSA3072.